src/args.rs

```diff
@@ -197,6 +197,10 @@ pub struct CliArgs {
     #[cfg(feature = "tls")]
     #[clap(long = "tls-key", requires = "tls-cert", value_hint = ValueHint::FilePath)]
     pub tls_key: Option<PathBuf>,
+
+    /// Enable readme redering in directories
+    #[clap(long = "readme")]
+    pub readme: bool,
```

Comment:
Please also generate an updated `README.md` for the repo and add this to the features list as it's a new cool feature.

Reply:
Ok. I'll do that.